x-pack/plugins/security_solution/server/lib/detection_engine/signals/enrichments/enrichment_by_type/host_risk.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { get, set, cloneDeep } from 'lodash';
+
+import { getHostRiskIndex } from '../../../../../../common/search_strategy/security_solution/risk_score/common';
+import { createSingleFieldMatchEnrichment } from '../create_single_field_match_enrichment';
+import type { CreateRiskEnrichment, GetIsRiskScoreAvailable } from '../types';
+
+export const getIsHostRiskScoreAvailable: GetIsRiskScoreAvailable = async ({
+  spaceId,
+  services,
+}) => {
+  const isHostRiskScoreIndexExist = await services.scopedClusterClient.asCurrentUser.indices.exists(
+    {
+      index: getHostRiskIndex(spaceId),
+    }
+  );
+
+  return isHostRiskScoreIndexExist;
+};
+
+export const createHostRiskEnrichments: CreateRiskEnrichment = async ({
+  services,
+  logger,
+  events,
+  spaceId,
+}) => {
+  return createSingleFieldMatchEnrichment({
+    name: 'Host Risk',
+    index: [getHostRiskIndex(spaceId)],
+    services,
+    logger,
+    events,
+    mappingField: {
+      eventField: 'host.name',
+      enrichmentField: 'host.name',
+    },
+    createEnrichmentFunction: (enrichment) => (event) => {
+      const risk = get(enrichment, `_source.risk`);
+      if (!risk) {
+        return event;
+      }
+      const newEvent = cloneDeep(event);
+      set(newEvent, '_source.host.risk.calculated_level', risk);
```

Comment:
Is this logic correct? We're setting whatever was in the `risk` field to `host.risk.calculated_level` ? Ditto for the user one.

Reply:
In this case, I just used, the current version of the host risk score, and when we update transforms regarding the new ECS RFC it will be different